### PR TITLE
[RDY] Adds location Mass Grave

### PR DESCRIPTION
**Pull request by Regilith**
#### Summary
```SUMMARY: Content "Adds new location 'Mass Grave'"```

#### Purpose of change
Planning on adding new "Left for Dead" Scenario, wake up naked, injured, and sick in a pile of dead bodies and zombies.

Also planning on adding variations, when I figure out how to do that.

Also adds good source of tainted meat for innawoods characters.

#### Describe the solution
Spawns pit of corpses with high zombie count, same spawning frequency as LMOE shelter.

I tested and it seems to be working fine. Might need some feedback on the amount of zombies spawned, it is high, but I could run out of the pit and away 4/4 times, and it *is* a pit full of dead bodies.

If there are any suggestions, let me know.

### CODE REVIEW

**Comment by ymber** [comment]
```
09:53:43 Formatted data/json/mapgen/mass_grave.json
09:53:43 Makefile:1051: recipe for target 'data/json/mapgen/mass_grave.json' failed
09:53:43 make: *** [data/json/mapgen/mass_grave.json] Error 1
```
Your new file needs linting.

**Comment by Regilith** [comment]
Now why is it failing...

**Comment by cainiaowu** [comment]
You can ignore the CI build, it has a tendency to randomly failing.

File: data/json/overmap_terrain.json [hunk 1]
```diff
@@ -9671,5 +9671,12 @@
     "name": "trailer park",
     "sym": 35,
     "color": "white"
+  },
+  {
+  "type": "overmap_terrain",
+  "id": "mass_grave",
+  "name": "Mass Grave",
```
Comment (Night-Pryanik) [inline data/json/overmap_terrain.json]: This should be lowercased.

**Pull request merged**